packages/types/src/api.ts

```diff
@@ -11,3 +11,40 @@ export const TraceListResponseSchema = z.array(TraceSummarySchema);
 export type TraceListResponse = z.infer<typeof TraceListResponseSchema>;
 
 export type TraceDetailSpansResponse = Array<OtelSpan>;
+
+export const CollectionSchema = z.object({
```

Comment:
Important! Gotta bump the version in `packages/types/package.json`

then edit the pull request description to remind us to publish the new version of fpx-types